ast/annotations_test.go

```diff
@@ -737,6 +768,78 @@ p = 1`,
 				},
 			},
 		},
+		{
+			note: "multiple subpackages, refs in rule heads", // NOTE(sr): same as above, but last module's rule is `foo.bar.p` in package `root`
```

Comment:
@johanfylling Does this test result look OK to you? It's less than the case above, but I think that's OK 🤔

Reply:
Yes, I believe this is what one would expect 👍

Reply:
Thanks!